tests/baselines/reference/jsdocImplements_class.js

```diff
@@ -73,7 +73,7 @@ declare class B3 implements A {
 }
 declare namespace Ns {
     export { C1 };
-    const C5: {
+    export const C5: {

```

Comment:
Yes, this was mistakenly missing an `export` modifier before. (The `export` statement above made it required, but the emitter never added it).